### DISCUSSION TITLE
EGSnrc 2021 has been released

### DISCUSSION THREAD

**Discussion by ftessier** [nrc-cnrc/EGSnrc#699] [Announcements]
**This is the 2021 release of EGSnrc.** Notable changes from the previous version are listed in the [release notes](https://github.com/nrc-cnrc/EGSnrc/wiki/Release-notes). Note that the recommended way to [cite EGSnrc](https://github.com/nrc-cnrc/EGSnrc/wiki/Citing-EGSnrc) has been updated. Please download the EGSnrc source code from the github repository, as per the  [installation instructions](https://github.com/nrc-cnrc/EGSnrc/wiki/Installation-overview) for your operating system. In addition, the assets below provide:

- the configuration GUIs for Linux and Windows;
- pre-compiled GUIs for Linux and Windows (egs_gui, egs_inprz, and egs_view);
- an archive of all the EGSnrc manuals available on http://nrc-cnrc.github.io/EGSnrc/; and 
- additional sample data sets in support of some EGSnrc examples. 

<hr /><em>This discussion was created from the release <a href='https://github.com/nrc-cnrc/EGSnrc/releases/tag/v2021'>EGSnrc 2021</a>.</em>

**Comment by rtownson** [comment]
There was a bug in the linux version of egs_view - it had a file path dependency. I just fixed this and updated the assets accordingly. Edit: not actually fixed yet...

**Comment by ProfLeao** [comment]


**Reply by ftessier** [reply]
Hi @ProfLeao, may I kindly ask that you repost your question under the *Question* topic, under [Discussions](https://github.com/nrc-cnrc/EGSnrc/discussions)?

**Reply by ProfLeao** [reply]
Ok! Sorry for the mistake.

**Reply by ftessier** [reply]
No worries, thank you!

**Comment by rtownson** [comment]
There is still a dependency issue in the linux pre-compiled egs_view for this release, I just haven't had time to troubleshoot it (my previous fix was a false positive).
